test/ads/icon/IconIntegrator.spec.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+describe("IconIntegrator", function () {
+
+  var VASTError = require('ads/vast/VASTError');
+  var IconIntegrator = require('ads/icon/IconIntegrator');
+  var Icon = require('ads/vast/Icon');
+
+  var xml = require('utils/xml');
+  var testUtils = require('../../test-utils');
+
+  function assertError(callback, msg, code) {
+    var error = testUtils.firstArg(callback);
+    assert.instanceOf(error, VASTError);
+    assert.equal(error.message, "VAST Error: " + msg);
+    if (code) {
+      assert.equal(error.code, code);
+    }
+  }
+
+  function createIcons() {
+    var xmlStr1 = '<Icon program="static1" width="80" height="80" xPosition="right" yPosition="bottom" offset="00:00:01" duration="00:00:02" apiFramework="string">' +
```

Comment:
you may wish to add support for string template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Good suggestion. Did it.